### PR TITLE
Add cds.write.archived.java.heap, ZGenerational to OpenJ9PropsExt

### PR DESCRIPTION
**Pull request by pshipton**
Issue https://github.com/eclipse-openj9/openj9/issues/17631

### CODE REVIEW

File: closed/test/jtreg-ext/requires/OpenJ9PropsExt.java [hunk 1]
```diff
@@ -54,7 +55,8 @@ public Map<String, String> call() {
             map.put("vm.musl", "false");
             map.put("vm.openj9", "true");
             map.put("vm.opt.final.ClassUnloading", "false");
-        }
+            map.put("vm.opt.final.ZGenerational", "false");
+       }
```
Comment (keithc-ca) [inline closed/test/jtreg-ext/requires/OpenJ9PropsExt.java:59]: Please undo the change to this line, unless you'd like to join the `catch` clause to it (but the indentation was correct).

Comment (pshipton) [reply closed/test/jtreg-ext/requires/OpenJ9PropsExt.java]: Updated

**Review by keithc-ca — APPROVED** [review]
